src/project/FileViewController.js

```diff
@@ -61,7 +61,6 @@ define(function (require, exports, module) {
      * @private
      */
     var _curDocChangedDueToMe = false;
-    var WORKING_SET_VIEW = "WorkingSetView";
```

Comment:
We can't do this.  You're assuming that this is only used internally in this file, but if you do a `git grep WORKING_SET_VIEW` you'll see that it's used in a lot of places.  This is a public API, and you have to maintain it for other call sites to not break.  When you make changes like this, *always* look at who else in the code uses it.  Don't assume that things are isolated to the particular case you're solving.

I've filed https://github.com/mozilla/brackets/issues/692 to deal with removing this code properly.